src/main/kotlin/com/openlattice/organization/OrganizationExternalDatabaseColumn.kt

```diff
@@ -0,0 +1,54 @@
+package com.openlattice.organization
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.authorization.securable.AbstractSecurableObject
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.client.serialization.SerializationConstants
+import org.apache.olingo.commons.api.edm.FullQualifiedName
+import java.util.*
+
+class OrganizationExternalDatabaseColumn
+
+/**
+ * Creates a securable object for an organization's entire database in Atlas
+ *
+ * @param id An optional UUID that will be automatically generated if not provided
+ * @param name The name of the column
+ * @param title A title for the object
+ * @param description A description for the object
+ * @param tableId The id of the table that contains this column
+ * @param organizationId The id of the organization that owns this column
+ */
+
+@JsonCreator
```

Comment:
It explicit constructor for JsonCreator annotation? IIRC correctly Kotlin module use the default constructor for serialization. Also KDoc can be moved above the class when using default constructor.

Reply:
Seems to work fine without the annotation so I'll remove it!